.eslintrc.yaml

```diff
@@ -0,0 +1,71 @@
+# This is a baseline set of rules intended for usage in any ES6-based project.
+#
+# Be sure to add your project's specific environment(s) as needed via external configuration.
+#   See: http://eslint.org/docs/user-guide/configuring#specifying-environments
+
+---
+rules: # http://eslint.org/docs/rules
+
+  # Possible Errors
+
+  valid-jsdoc: [1]
+  curly: [2]
+  dot-notation: [2]
```

Comment:
I'm not sure about this one; sometimes source actually reads much better when it's uniform, which means that a function in which one property *has* to be looked up using "array" notation reads much nicer if the other similar accesses also use array notation. I'd personally leave this off.